webpack.config.js

```diff
@@ -29,6 +31,11 @@ module.exports = {
       },
     ],
   },
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env.NODE_ENV': JSON.stringify('production')
```

Comment:
I think in development this plugins should not be there. Dunno gotta merge it and see how it works.

Reply:
UMD should not require build step, since it can be used in browsers. So `NODE_ENV` should be either `production` or `development`. Currently webpack adds process polyfill which sets `NODE_ENV` as `undefined`, so it's not `production`. But you excluded `prop-types` from umd and `prop-types` definitions are fails in runtime.